src/visualization/reducers/impression.ts

```diff
@@ -3,6 +3,11 @@ import { Types } from "../actions";
 export default function (state = null, action) {
     switch (action.type) {
         case Types.SelectSession:
+            for (var payload of action.payload) {
```

Comment:
>payload [](start = 39, length = 7)

also a nit, but payload is being used enough that I think this code has become confusing. Action can either have a payload in it, or a payload of payloads? Can we disambiguate the names slightly?